pygeoapi/process/manager/base.py

```diff
@@ -194,13 +200,15 @@ def _execute_handler_async(self, p: BaseProcessor, job_id: str,
         """
         _process = dummy.Process(
             target=self._execute_handler_sync,
-            args=(p, job_id, data_dict)
+            args=(p, job_id, data_dict, subscriber)
         )
         _process.start()
         return 'application/json', None, JobStatus.accepted
 
     def _execute_handler_sync(self, p: BaseProcessor, job_id: str,
-                              data_dict: dict) -> Tuple[str, Any, JobStatus]:
+                              data_dict: dict,
+                              subscriber: Optional[Subscriber] = None,
```

Comment:
Missing description of `subscriber` parameter.

Reply:
I don't have the permissions to push the repo directly and not sure if this warrants a PR, would you mind adding something like this line to the sync and async execute methods?
```:param subscriber: optional `Subscriber` specifying callback URLs ```

Reply:
I'll push a fix.